client/dashboard/domains/domain-overview/transferred-domain-details.tsx

```diff
@@ -0,0 +1,146 @@
+import {
+	Button,
+	__experimentalText as Text,
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+} from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import Notice from '../../components/notice';
+
+export default function TransferredDomainDetails( { domain }: { domain: Domain } ) {
+	const {
+		current_user_is_owner,
+		last_transfer_error,
+		domain: domain_name,
+		owner,
+		transfer_status,
+	} = domain;
+
+	const renderDescriptionText = () => {
+		if ( last_transfer_error ) {
+			return current_user_is_owner ? (
+				<>
+					<Text as="p">
+						{ createInterpolateElement(
+							__(
+								/* translators: <domain/> is the domain name */
+								'We tried to start a transfer for your domain <domain/> but we got the following error:'
+							),
+							{
+								domain: <strong>{ domain_name }</strong>,
+							}
+						) }
+					</Text>
+					<Text as="p">
+						<code>{ last_transfer_error }</code>
+					</Text>
+					<Text as="p">
+						{ __(
+							'Please restart the transfer or contact your current domain provider for more details.'
+						) }
+					</Text>
+				</>
+			) : (
+				<Text as="p">
+					{ createInterpolateElement(
+						__(
+							/* translators: <domain/> is the domain name, <owner/> is the domain owner */
+							'We tried to start a transfer for your domain <domain/> but an error occurred. Please contact the domain owner, <owner/>, for more details.'
+						),
+						{
+							domain: <strong>{ domain_name }</strong>,
+							owner: <strong>{ owner }</strong>,
+						}
+					) }
+				</Text>
+			);
+		}
+
+		if ( DomainTransferStatus.PENDING_START === transfer_status ) {
+			return current_user_is_owner ? (
+				<Text as="p">
+					{ createInterpolateElement(
+						__(
+							/* translators: <domain/> is the domain name */
+							'We need you to complete a couple of steps before we can transfer <domain/> from your current domain provider to WordPress.com. Your domain will stay at your current provider until the transfer is completed.'
+						),
+						{
+							domain: <strong>{ domain_name }</strong>,
+						}
+					) }
+				</Text>
+			) : (
+				<Text as="p">
+					{ createInterpolateElement(
+						__(
+							/* translators: <owner/> is the domain owner */
+							'This domain transfer is waiting to be initiated. Please contact the domain owner, <owner/>, to start it.'
+						),
+						{
+							owner: <strong>{ owner }</strong>,
+						}
+					) }
+				</Text>
+			);
+		} else if ( DomainTransferStatus.CANCELLED === transfer_status ) {
+			return current_user_is_owner ? (
+				<Text as="p">
+					{ createInterpolateElement(
+						__(
+							/* translators: <domain/> is the domain name */
+							'We were unable to complete the transfer of <domain/>. You can remove the transfer from your account or try to start the transfer again.'
```

Comment:
```suggestion
							'We were unable to complete the transfer of <domain/>. You can remove the transfer from your account or try to start it again.'
```

Does it sound better when we remove the "transfer" repetition?